eogrow/core/pipeline.py

```diff
@@ -169,26 +163,17 @@ def run_execution(
             self.patch_list will be used
         :return: Lists of successfully/unsuccessfully executed EOPatch names and the result of the EOWorkflow execution
         """
-        executor_class: type[EOExecutor]
+        ray.init(address="auto", ignore_reinit_error=True)
 
-        execution_kind = self._init_processing()
-        extra_kwargs = {}
-        if execution_kind is ProcessingType.RAY:
-            executor_class = RayExecutor
-            extra_kwargs = {"ray_remote_kwargs": self.config.ray_remote_kwargs}
-        else:
-            executor_class = EOExecutor
-            executor_run_params["workers"] = self.config.workers
-
-        LOGGER.info("Starting %s for %d EOPatches", executor_class.__name__, len(execution_kwargs))
+        LOGGER.info("Starting processing for %d EOPatches", len(execution_kwargs))
 
         # Unpacking manually to ensure order matches
         list_of_kwargs, execution_names = [], []
         for exec_name, exec_kwargs in execution_kwargs.items():
             list_of_kwargs.append(exec_kwargs)
             execution_names.append(exec_name)
 
-        executor = executor_class(
+        executor = RayExecutor(
```

Comment:
One last concern I have is with debugging, currently we could run the pipeline locally and just debug using the ipdb/vscode debugger, but this might cause problems now since it's wrapped in ray.

There are some mentions that for local ray clusters the behavior should be fine, or that one can use the [ray debugger](https://docs.ray.io/en/latest/ray-observability/user-guides/debug-apps/ray-debugging.html) but I didn't manage to stop inside an eogrow pipeline test, which is worrying.

Reply:
the debugging now works for me after the recent changes. I think it's good enough for now.